ref/net46/Microsoft.Build.Utilities.Core/Microsoft.Build.Utilities.Core.cs

```diff
@@ -314,7 +314,8 @@ public enum TargetDotNetFrameworkVersion
         Version46 = 7,
         Version461 = 8,
         Version462 = 10,
-        VersionLatest = 10,
+        Version47 = 11,
+        VersionLatest = 11,
```

Comment:
You're making a breaking change in an in-place update.

How this broke Roslyn last time, was that we installed VS 2015 Update 2 onto the build machine - not realizing that this would cause us to pick up a new value of the enum for VersionLatest. This would run on another machine without this value and we were broken.